Examples/PublisherExample/PublisherExample.xcodeproj/project.pbxproj

```diff
@@ -497,7 +504,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = C4YGD8Z6G2;
```

Comment:
Was this change intentional? Don't we want to keep the development team empty and let automatic code signing do its thing?

Reply:
@lukasz-szyszkowski did you have thoughts on this?

Reply:
here 633c0d8fdfde410438641a29cf9cd01edbbbbcf5

Reply:
@lawrence-forooghian Could you resolve this conversation, please?

Reply:
Done!